src/machine/board_stm32.go

```diff
@@ -1,16 +1,5 @@
-// +build bluepill nucleof103rb stm32f4disco
+// +build bluepill nucleof103rb stm32f4
```

Comment:
I think this file should be deleted.

Reply:
Unless there is some functionality that is shared across all boards in this family? Seems like we can leave it in for now.